cmd/telemeter-server/main.go

```diff
@@ -382,9 +408,10 @@ func (o *Options) Run() error {
 		internalProtected.Handle("/debug/cluster", c)
 	}
 
-	var transforms metricfamily.Transformer
+	transforms := metricfamily.MultiTransformer{}
+	transforms.With(whitelister)
 	if len(o.Labels) > 0 {
-		transforms = metricfamily.NewLabel(o.Labels, nil)
+		transforms.With(metricfamily.NewLabel(o.Labels, nil))
```

Comment:
Unrelated and not to ourselves: as far as I see, the Label transformer is modifying its state at every http request in an unsynchronized way:

https://github.com/openshift/telemeter/blob/6d3dc6367a168fce7e5a9ea9f1a43240c7f2dc96/pkg/metricfamily/label.go#L36

We should synchronize those modifications in the label transformer.